skyvern/webeye/utils/dom.py

```diff
@@ -294,7 +295,7 @@ def get_tag_name(self) -> str:
         return self.__static_element.get("tagName", "")
```

Comment:
shall we apply the same pattern to all these getters?

Reply:
made the updates